Slim/Player/Squeezebox1.pm

```diff
@@ -142,17 +142,17 @@ sub isReadyToStream {
 		# Bug 15490, work out whether or not we can gaplessly stream PCM/AIFF. Assume that if
 		# channels, samplesize and samplerate all match, we'll be fine.
 		if ( $CSF eq 'pcm' || $CSF eq 'aif' ) {
-			if ( $playingSong ) {
-				if (   $playingSong->channels   == $song->channels
-					&& $playingSong->samplesize == $song->samplesize
-					&& $playingSong->samplerate == $song->samplerate
-				) {
-					return 1;
-				}
-			}
-		}
-	}
-	
+                        if ( $playingSong ) {
```

Comment:
Could you please revert whitespace changes? Make sure you're using tabs rather than spaces.

Reply:
You should probably get the two `currentTrack` objects beforehand and check their existence, too.

```perl
my $playingTrack = $playingSong->currentTrack;
my $songTrack = $song->currentTrack;

if ($playingTrack && $songTrack && $playingTrack->channels == $songTrack->channels && ...
```